java/src/jmri/jmrit/jython/JynstrumentFactory.java

```diff
@@ -33,6 +33,10 @@ public static Jynstrument createInstrument(String path, Object context) {
         }
         String jyFile = path + File.separator + className + ".py";
         ScriptEngine engine = JmriScriptEngineManager.getDefault().getEngine(JmriScriptEngineManager.PYTHON);
+        if (engine==null){
```

Comment:
Please use a SuppressWarning annotation here—if someone has removed parts of a JMRI installation that we can’t compile without, we should crash on the NPE (or whatever error the JmriScriptEngineManager will throw because missing dependencies are not present instead of logging here.

Reply:
Thanks, changed to annotation.